RecoLocalTracker/SiPixelRecHits/plugins/PixelCPEClusterRepairESProducer.cc

```diff
@@ -52,11 +53,15 @@ PixelCPEClusterRepairESProducer::PixelCPEClusterRepairESProducer(const edm::Para
   hTTToken_ = c.consumes();
   templateDBobjectToken_ = c.consumes();
   templateDBobject2DToken_ = c.consumes();
-  if (DoLorentz_) {
+
+  char const* laLabel = "";  // standard LA, from calibration, label=""
+  lorentzAngleToken_ = c.consumes(edm::ESInputTag("", laLabel));
+
+  if (doLorentzFromAlignment_) {
     lorentzAngleToken_ = c.consumes(edm::ESInputTag("", "fromAlignment"));
```

Comment:
```suggestion
  if (useLAFromDB_ || doLorentzFromAlignment_) {
    char const* laLabel = doLorentzFromAlignment_ ? "fromAlignment" : "";
    lorentzAngleToken_ = c.consumes(edm::ESInputTag("", laLabel));
```

this is matching the actual use case in `::produce` (only one payload is actually consumed; the current implementation can declare two different consumes). However I'm not sure what your actual goal is for the case with both flags set to true.

Reply:
Ok I updated to your suggestion